tools/android/build_incremental_dexmanifest.py

```diff
@@ -32,7 +32,7 @@
 
 import hashlib
 import os
-from queue import Queue
+from Queue import Queue
```

Comment:
Should we use

try:
    import Queue from Queue
except ImportError:
    import queue as Queue 
here to support both python 2 and 3?

Reply:
```python
try:
    from Queue import Queue
except ImportError:
    import queue as Queue 
```

This is the syntax that worked when python 2 is used.

Reply:
@ThomasCJY can you change the code to the alternative suggested by @arunkumar9t2 ? That'll make this work in both Pyton2 and 3.

Reply:
updated

Reply:
The change looks reasonable to me. Unfortunately I lack the necessary permissions to stamp the diff. I think @ahumesky is the right person to loop in here?